src/datepicker/datepicker.spec.ts

```diff
@@ -442,6 +442,18 @@ describe('ngb-datepicker', () => {
     expect(document.activeElement).toBe(datepicker.nativeElement);
   });
 
+  it('should emit select event when select date', () => {
```

Comment:
You should also add the tests for:

- clicking on the same date twice
- pressing Enter key on the same date twice